src/document-submit.js

```diff
@@ -64,7 +67,7 @@ export function onDocumentFormSubmit_(e) {
     user().assert(action,
         'form action attribute is required for method=GET: %s', form);
     assertHttpsUrl(action, dev().assertElement(form), 'action');
-    user().assert(!startsWith(action, urls.cdn),
+    user().assert(!isProxyOrigin(action),
```

Comment:
The modified `isProxyOrigin` return true for `https://localhost:8000/c/abc.html`, also return true for `https://localhost:8000/v/abc.html` but return false for `https://localhost:8000/abc.html`. So `!isProxyOrigin` is not blocking normal `localhost`.
@mkhatib About localhost case, when do we want to proceed?